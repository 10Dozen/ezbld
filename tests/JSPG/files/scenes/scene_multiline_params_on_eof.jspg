# SceneMultilineParamsOnEOF

Line 1.

Line ${Foo.Bar()}.

**post_exec:
    JSPG.GoTo('NormalScene')