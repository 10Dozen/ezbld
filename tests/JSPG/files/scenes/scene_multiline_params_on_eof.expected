[0]
cls = JSPGScene
name = SceneMultilineParamsOnEOF
post_exec =
    \tJSPG.GoTo('NormalScene')

desc =
    Line 1.

    Line ${Foo.Bar()}.
